examples/resources/awscc_elasticbeanstalk_application/elasticbeanstalk_application.tf

```
@@ -0,0 +1,14 @@
+resource "awscc_elasticbeanstalk_application" "example_elasticbeanstalk_application" {
```

```suggestion
resource "awscc_elasticbeanstalk_application" "example" {
```

simplify resource name